movieHW/movieHW/Views/DetailViewController.swift

```diff
@@ -0,0 +1,161 @@
+// DetailViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Экран детального описания фильма
+final class DetailViewController: UIViewController {
+    // MARK: - Constants
+
+    private enum Constants {
+        static let defaultBlack = "defaultBlack"
+        static let defaultOrange = "defaultOrange"
+        static let watch = "Смотреть"
+        static let shareButtonImageName = "square.and.arrow.up"
+        static let clearString = ""
+    }
+
+    // MARK: Private Visual Components
+
+    private var movieImageView = UIImageView()
```

Comment:
```suggestion
    private let movieImageView = UIImageView()
```
меняем var на let вездее, где позволяет компилятор